grpc-ballerina/grpc_errors.bal

```diff
@@ -81,7 +81,7 @@ public type DataMismatchError distinct Error;
 public type ClientAuthError distinct Error;
 
 # Represents an error when reaching the end of the client stream.
-public type EOS distinct Error;
+public type EosError distinct Error;
```

Comment:
we can remove this error type now. we are not using anywhere

Reply:
Fixed in here https://github.com/ballerina-platform/module-ballerina-grpc/pull/154/commits/0203e25d8289ee970a44df37dd233c69f099dac1